sci-astronomy/scamp/scamp-2.10.0.ebuild

```
@@ -0,0 +1,50 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+DESCRIPTION="Astrometric and photometric solutions for astronomical images"
+HOMEPAGE="http://www.astromatic.net/software/scamp"
```

Sorry for the late request, but would you mind adding a repository url here? Thanks!
```suggestion
HOMEPAGE="http://www.astromatic.net/software/scamp https://github.com/astromatic/scamp"
```

---

No problem. Fixed ;)